bigtable-hbase-1.x-parent/bigtable-hbase-1.x-shaded/pom.xml

```diff
@@ -130,10 +130,37 @@ limitations under the License.
       <version>${grpc-conscrypt.version}</version>
       <scope>runtime</scope>
     </dependency>
+    <!--needs to be declared as a dependency to be excluded from shading-->
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>jsr305</artifactId>
+      <version>${jsr305.version}</version>
+    </dependency>
   </dependencies>
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>license-maven-plugin</artifactId>
+        <version>2.0.0</version>
+        <executions>
+          <execution>
+            <id>default-cli</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>add-third-party</goal>
+            </goals>
+            <configuration>
+              <excludedScopes>test,provided,system</excludedScopes>
+              <excludedArtifacts>jsr305|metrics-core|commons-logging|hbase-shaded-client|slf4j-api|slf4j-log4j12|log4j|htrace-core|findbugs-annotations|conscrypt-openjdk-uber</excludedArtifacts>
+              <excludedGroups>org.bouncycastle|org.checkerframework|javax.annotation</excludedGroups>
```

Comment:
please add a comment this should follow the exclude rules of shade plugin and please order the exclusion in the same to make it easier to compare